### DISCUSSION TITLE
Open Source LLM

### DISCUSSION THREAD

**Discussion by JannikSchneider12** [Future-House/paper-qa#180] [General]
Hey,

Did someone already tried out an open LLM like Llama2 and can say something about the performance?

**Comment by limcheekin** [comment]
For Retrieval Augmented Generation (RAG) use case, I think the quality of the generate text is good enough, though I didn't try it with paper-qa.

**Comment by Raul-dot-valdez** [comment]
HI Jannik, you can watch this video: [https://www.youtube.com/watch?v=k2FHUP0krqg](url)

[https://gist.github.com/Raul-dot-valdez/663e378775192b50b4254f9d6abd6fd2](url)

**Comment by jamesbraza** [comment]
Hi all, let's consolidate discussion here: https://github.com/Future-House/paper-qa/issues/378
